### PR TITLE
Update CHANGES.md for edge-19.1.4 release

### PR DESCRIPTION
**Pull request by alpeb**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+    when injecting proxies (more details at https://linkerd.io/2/cni)
+  * Added more improvements to the API for `ListPods` (thanks @alenkacz!)
+
+* Web UI
+  * Grayed-out the tap icon for requests from sources that are not meshed
+
+* CLI
+  * Added the `--proto` flag to `linkerd profile` to output a service profile
+    based on a Protobuf spec file -- Retryable methods can be declared in the Protobuf
+    file using the `@Retryable` annotation
```
Comment (adleong) [inline CHANGES.md]: I don't think this use of the `@Retryable` annotation is implemented

Comment (alpeb) [reply CHANGES.md]: oh ok, I'll remove that :+1:

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
+## edge-19.1.4
+
+* Controller
+  * Added timeouts support! Configurable in the service profiles for each route
+  * Added an experimental CNI plugin to avoid requiring the NET_ADMIN capability
+    when injecting proxies (more details at https://linkerd.io/2/cni)
+  * Added more improvements to the API for `ListPods` (thanks @alenkacz!)
+
```
Comment (klingerf) [inline CHANGES.md]: We typically don't include a newline between top-level bullets. Mind removing this?

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+* Web UI
+  * Grayed-out the tap icon for requests from sources that are not meshed
+
```
Comment (klingerf) [inline CHANGES.md]: And please remove this newline as well.

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,23 @@
+## edge-19.1.4
+
+* Controller
+  * Added timeouts support! Configurable in the service profiles for each route
```
Comment (klingerf) [inline CHANGES.md]: Can we reword this as either "Added timeout support!" or "Added support for timeouts!"?

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+  * Added an experimental CNI plugin to avoid requiring the NET_ADMIN capability
+    when injecting proxies (more details at https://linkerd.io/2/cni)
```
Comment (klingerf) [inline CHANGES.md]: We should give `@codeman9` a shoutout here for all of his hard work.

File: CHANGES.md (hunk 4 above)
Comment (alpeb) [reply CHANGES.md]: sure

File: CHANGES.md (hunk 5 above)
Comment (alpeb) [reply CHANGES.md]: will do
